apps/.storybook/config.js

```diff
@@ -13,21 +14,27 @@ import '../style/netsim/style.scss';
 import '../style/applab/style.scss';
 import '../src/templates/GameButtons.story.scss';
 
+if (!addons.getChannel()) {
+  // Provide a mock channel to prevent "Accessing nonexistent addons channel"
+  // errors.
+  addons.setChannel(mockChannel());
+}
+
```

Comment:
My previous Drone build failed with this error: 
<img width="618" alt="Screen Shot 2020-03-05 at 8 27 01 PM" src="https://user-images.githubusercontent.com/12300669/76051507-e5aa7580-5f1f-11ea-83ea-d02cf1f0a8c1.png">

Since I'm calling `setSelectedLessons` in `componentWillMount` in `LessonStatusList` when I use actions in the `LessonStatusList.story` this error occurs.  I followed the advice in the "Why is there no addons channel ?" section of [storybook.js.org/docs/basics/faq/](storybook.js.org/docs/basics/faq/). 

All other changes in this file are from prettier auto-formatting.